docs/markdown_code_symlinks.py

```diff
@@ -1,7 +1,18 @@
 import logging
```

Comment:
It would be really good if we could add some tests for this code in some way?

Reply:
Agreed, I will think on how to perform these tests.
I suppose that docutils or sphinx itself has some kind of check on whether links are broken.

So a possible test could be to produce a doc build and check that every link references to a valid destination.